doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
```

Comment:
Could we add another row called technology where we state the technology which the example is presenting. Like for instance:
- WaitSet
- Listener
- n:m Communication
- IPC
- InterThreadCommunication
- Introspection

Reply:
Either lexicographically or by "complexity" which is very subjective (so probably not a good choice). The technology category column (maybe we might need a better name) would be helpful I guess.

Reply:
With such a small number of examples, I currently don't see the benefit of a technology column. E.g. adding "WaitSet" and "Introspection" are just duplicates